documentation/printing.md

```diff
@@ -0,0 +1,25 @@
+# 3D Printing
+
+{{BOM}}
+
+[PLA]:parts/filament.md "{cat:consumable}"
+
+## Print the custom components {pagestep}
+
+
+Using the slicer that comes with your [3d printer](parts/3dprinter.md){qty:1, cat: tool}, slice thent the following parts, and then print them.
+
+Some parts require support. A brim is advised for all parts.
```

Comment:
Are the 3D prints that require supports marked somehow, or do the users need to use their best judgement?

Reply:
Yeah. We can mark them in the YAML when we explain the part. The table defaults to saying no if nothing is set.

See https://julianstirling.github.io/nimble-example-documentation/printing.html and https://github.com/Wakoma/nimble-example-documentation/pull/1